$NetBSD: patch-CMakeLists.txt,v 1.1 2020/11/29 22:39:10 dbj Exp $

Fix build on case insensitive build dirs where a created
cmake configuration directory conflicts with the output binary.

--- CMakeLists.txt.orig	2019-06-29 00:19:28.000000000 +0000
+++ CMakeLists.txt
@@ -626,18 +626,18 @@ install(TARGETS ${qhull_TARGETS_INSTALL}
 include(CMakePackageConfigHelpers)
 
 write_basic_package_version_file(
-    "${CMAKE_CURRENT_BINARY_DIR}/Qhull/QhullConfigVersion.cmake"
+    "${CMAKE_CURRENT_BINARY_DIR}/Qhull_cmake/QhullConfigVersion.cmake"
     VERSION ${qhull_VERSION}
     COMPATIBILITY AnyNewerVersion
 )
 
 export(EXPORT QhullTargets
-  FILE "${CMAKE_CURRENT_BINARY_DIR}/Qhull/QhullTargets.cmake"
+  FILE "${CMAKE_CURRENT_BINARY_DIR}/Qhull_cmake/QhullTargets.cmake"
   NAMESPACE Qhull::
 )
 
 configure_file(${PROJECT_SOURCE_DIR}/build/config.cmake.in
-  "${CMAKE_CURRENT_BINARY_DIR}/Qhull/QhullConfig.cmake"
+  "${CMAKE_CURRENT_BINARY_DIR}/Qhull_cmake/QhullConfig.cmake"
   @ONLY
 )
 
@@ -652,8 +652,8 @@ install(EXPORT QhullTargets
 )
 install(
   FILES
-    "${CMAKE_CURRENT_BINARY_DIR}/Qhull/QhullConfig.cmake"
-    "${CMAKE_CURRENT_BINARY_DIR}/Qhull/QhullConfigVersion.cmake"
+    "${CMAKE_CURRENT_BINARY_DIR}/Qhull_cmake/QhullConfig.cmake"
+    "${CMAKE_CURRENT_BINARY_DIR}/Qhull_cmake/QhullConfigVersion.cmake"
   DESTINATION
     ${ConfigPackageLocation}
   COMPONENT
